### PR TITLE
Add callback log througput

### PR DESCRIPTION
**Pull request by HDThang**
# What does this PR do?
Add mlflow and log num param to train script of these taks:
- speech pretraining 
- speech recognition
- contrastive-image-text

### CODE REVIEW

File: examples/pytorch/contrastive-image-text/run_clip.py [hunk 1]
```diff
@@ -500,6 +537,12 @@ def filter_corrupt_images(examples):
         eval_dataset=eval_dataset if training_args.do_eval else None,
         data_collator=collate_fn,
     )
+    trainer.add_callback(TBTrainerCallback)
+    # Mlflow initial
+    #set the os enviroment for MLflowCallback
+    os.environ["DISABLE_MLFLOW_INTEGRATION"] = "False"
+    os.environ["HF_MLFLOW_LOG_ARTIFACTS"]="False"
```
Comment (yeomni) [inline examples/pytorch/contrastive-image-text/run_clip.py:544]: Do we have no problem getting all the MLFlow logs in the end even when it's false?

File: examples/pytorch/speech-pretraining/train_wav2vec2_pretraining_no_trainer.sh
Comment (yeomni) [inline examples/pytorch/speech-pretraining/train_wav2vec2_pretraining_no_trainer.sh:1]: Why do we have such shell script? I've thought we are going to run all the models with Combination Runner. We should rather add additional models to the existing yaml config file to run such things.

**Review by yeomni — CHANGES_REQUESTED** [review]
We need updates on config files to run additional models, not bash script.

File: examples/pytorch/contrastive-image-text/run_clip.py (hunk 1 above)
Comment (HDThang) [reply examples/pytorch/contrastive-image-text/run_clip.py:544]: The default option in the trainer is False, so there are no problems if we set them to False here. I just put them there to remind that we have the options.

File: examples/pytorch/speech-pretraining/train_wav2vec2_pretraining_no_trainer.sh
Comment (HDThang) [reply examples/pytorch/speech-pretraining/train_wav2vec2_pretraining_no_trainer.sh:1]: I'll remove it. It looks like I've added it accidentally.

**Review by yeomni — APPROVED** [review]
LGTM

**Pull request merged**